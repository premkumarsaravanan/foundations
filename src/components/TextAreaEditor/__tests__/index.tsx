import * as React from 'react'
import { shallow } from 'enzyme'
import { TextAreaEditor, TextAreaEditorProps } from '../index'
import toJson from 'enzyme-to-json'

const props: TextAreaEditorProps = {
  id: 'username',
  name: 'username',
  labelText: 'User name',
  placeholder: 'enter your name here'
}

describe('TextAreaEditor', () => {
  it('should match a snapshot', () => {
    expect(toJson(shallow(<TextAreaEditor {...props} />))).toMatchSnapshot()
  })
})
