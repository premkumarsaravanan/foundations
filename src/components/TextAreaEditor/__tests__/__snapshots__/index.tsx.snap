// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextAreaEditor should match a snapshot 1`] = `
<FormikConnect(FieldInner)
  name="username"
  render={[Function]}
/>
`;
